### DISCUSSION TITLE
使用sealos升级集群时，是否可以更换网络组件？

### DISCUSSION THREAD

**Discussion by Chris0408** [labring/sealos#3397] [Q&A]
之前已经使用如下命令: `sealos run labring/kubernetes-docker:v1.24.8-4.1.5 labring/helm:v3.8.2 labring/calico:v3.24.1` 部署了集群，现在想在升级集群的同时将网络组件更换为flannel是否支持这种操作？

**Comment by cuisongliu** [comment]
需要手动升级，可能需要下掉calico再升级flannel

**Comment by cuisongliu** [comment]
这种操作不建议，所有的pod都需要重新加载网络了。
